swap_meet/vendor.py

```diff
@@ -1,2 +1,113 @@
 class Vendor:
-    pass
\ No newline at end of file
+    """
+    A class to represent a vendor, managing the inventory and providing
+    methods for adding, removing, and swapping items between vendors.
+    """
+
+    def __init__(self, inventory=None):
+        """
+        Initializes a Vendor instance with an optional inventory list.
+        If no inventory is provided, an empty list is initialized.
+        """
+        self.inventory = inventory or []
```

Comment:
👍 Short circuiting to assign default values like you do here works great. Note that short-circuit evaluation only works when you want to execute a certain piece of code if a condition is True or False. 

You could also use a ternary here if you wanted, either works. 

What the  ternary operator offers that short-circuit evaluation doesn't is that you can evaluate a more complex conditional statement. Like:
```py
some_var = "a value if True" if x % 2 == 0 else "a value if False"